Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +28,7 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (!IsArrayNullOrEmpty(array)) Array.Reverse(array);
```

Comment:
I kind of assumed that people would manually implement it. So please do a mnual implementation of a reverse 😄